modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ if (CONFIG_BUILD_WITH_TFM)
     set(BL2_ELF_FILE ${TFM_BINARY_DIR}/bin/bl2.elf)
     set(BL2_BIN_FILE ${TFM_BINARY_DIR}/bin/bl2.bin)
     set(BL2_HEX_FILE ${TFM_BINARY_DIR}/bin/bl2.hex)
+    if(CONFIG_TFM_BL1)
+      set(BL2_SIGNED_BIN_FILE ${TFM_BINARY_DIR}/bin/bl2_signed.bin)
```

Comment:
I am just curious here but do we need to have both the BL2_[HEX/BIN]_FILE and the BL2_SIGNED_[HEX/BIN]_FILE set at the same time?

I am asking because if it is possible to just set the value of the  BL2_HEX_FILE for example to have a different path depending on the CONFIG_TFM_BL1 configuration it will need less changes in this file

Reply:
Ok, understood, thank you for the explanation